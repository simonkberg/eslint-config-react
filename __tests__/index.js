/* eslint-env jasmine */

const eslintrc = require('../')
const CLIEngine = require('eslint').CLIEngine

describe('@simonkberg/eslint-config-react', () => {
  it('should match the snapshot', () => {
    expect(eslintrc).toMatchSnapshot()
  })

  it('should load config in eslint', () => {
    const cli = new CLIEngine({
      useEslintrc: false,
      configFile: 'eslintrc.json',
    })

    const stub = 'const foo = 1\nconst bar = baz => baz\nbar(foo)\n'
    const result = cli.executeOnText(stub)

    expect(result.errorCount).toEqual(0)
    expect(result.warningCount).toEqual(0)
  })
})
