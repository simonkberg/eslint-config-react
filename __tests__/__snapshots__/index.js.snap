// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@simonkberg/eslint-config-react should match the snapshot 1`] = `
Object {
  "extends": Array [
    "@simonkberg/eslint-config",
    "standard-react",
  ],
}
`;
